assets/airports/kjfk.json

```diff
@@ -7,227 +8,565 @@
+                "GREKI": [],
+                "JUDDS": ["JUDDS"]
             },
-            "draw": [["CRI", "OGY"], ["CRI", "_CRI3"], ["_RWY4L3", "_RWY4L3to100"], ["JFK", "SAX*"], ["JFK", "COATE*"], ["JFK", "NEION*"], ["JFK","HAAYS*"], ["JFK", "GAYEL*"], ["JFK", "GREKI*"], ["JFK", "MERIT*"], ["JFK", "BAYYS*"], ["JFK", "BDR*"], ["JFK", "BETTE*"], ["JFK", "HAPIE*"], ["JFK", "SHIPP*"], ["JFK", "WAVEY*"], ["JFK", "WHITE*"], ["JFK", "DIXIE*"], ["JFK", "RNGRR*"], ["JFK", "RBV*"], ["JFK", "ARD*"], ["JFK", "SBJ*"], ["JFK", "LANNA*"]]
+            "draw": [
+                ["GREKI*", "JUDDS*"]
+            ]
         },
-        "DEEZZ4": {
-            "icao": "DEEZZ4",
-            "name": "DEEZZ FOUR",
+        "HAPIE6": {
+            "icao": "HAPIE6",
+            "name": "Happy Six",
             "rwy": {
-                "31L": ["_315I238SKORR", "SKORR", ["CESID", "A25+"], "YNKEE"],
-                "31R": ["_315I243SKORR","SKORR", ["CESID", "A25+"], "YNKEE"],
-                "13R": [["_RWY13R3DME", "A5.2+"]],
-                "13L": [["_RWY13L3DME", "A5.2+"]],
-                "4R" : ["_RWY4L3"],
-                "4L" : ["_RWY4L3"],
-                "22R": ["YNKEE"],
-                "22L": ["YNKEE"]
+                "KJFK4L" : ["#099"],
+                "KJFK4R" : ["#099"],
+                "KJFK13L": ["#134"],
+                "KJFK13R": ["#134"],
+                "KJFK22L": ["#224"],
+                "KJFK22R": ["#224"],
+                "KJFK31L": ["CRI", ["_CRI176002", "A25+"], "#176"],
+                "KJFK31R": ["CRI", ["_CRI176002", "A25+"], "#176"]
             },
+            "body": [],
             "exitPoints": {
-                "CANDR": ["DEEZZ", ["HEERO", "A50+"], "KURNL", "CANDR"],
-                "TOWIN": ["DEEZZ", ["HEERO", "A50+"], "KURNL", "TOWIN"]
+                "HAPIE": ["HAPIE"]
             },
-            "draw": [["SKORR", "CESID", "YNKEE"], ["DEEZZ", "HEERO", "KURNL", "CANDR*"], ["DEEZZ", "HEERO", "KURNL", "TOWIN*"]]
+            "draw": [
+                ["HAPIE*"]
+            ]
         },
-        "SKORR3": {
-            "icao": "SKORR3",
-            "name": "SKORR THREE",
+        "MERIT6": {
+            "icao": "MERIT6",
+            "name": "Merit Six",
             "rwy": {
-                "31R": ["_315I243SKORR"],
-                "31L": ["_315I238SKORR"]
+                "KJFK4L" : ["#099"],
+                "KJFK4R" : ["#099"],
+                "KJFK13L": ["#134"],
+                "KJFK13R": ["#134"],
+                "KJFK22L": ["#224"],
+                "KJFK22R": ["#224"],
+                "KJFK31L": ["CRI", ["_CRI176002", "A25+"], "#176"],
+                "KJFK31R": ["CRI", ["_CRI176002", "A25+"], "#176"]
             },
+            "body": ["MERIT"],
             "exitPoints": {
-                "RNGRR": ["SKORR", ["METSS", "A25+"], "RNGRR"],
-                "YNKEE": ["SKORR", ["CESID", "A25+"], "YNKEE"]
+                "MERIT": [],
+                "PUT": ["PUT"]
```

Comment:
Line 419: I believe it goes MERIT then HFD then PUT

Reply:
huh! good catch... done